### DISCUSSION TITLE
How about we make an android app?

### DISCUSSION THREAD

**Discussion by harshilishere** [filebrowser/filebrowser#1477] [Ideas]
Let's say I have a server running and wanna access files easily without the feel of opening it in some mobile browser(or making some sort of cheap web app). Are there backend apis which will let me fetch just the data? I am willing to make a mobile application in flutter which will let the user access there own server. Also what is the possibility to stream the media contents? Would like to play songs. Currently it downloads the songs and then play it.

**Comment by mustafaarif** [comment]
Have you looked into Webview for flutter? That can be an easy implementation to view filebrowser in app.

**Reply by harshilishere** [reply]
Seems a practical and quick approach as of now. But I was thinking something like making an app for let's say viewing movies. i.e.:
rn the movie files play in webbrowser or you have to copy/paste link into vlc with jwt token. An app on the other hand can be embedded with some good player and movies can be played from there.
Similarly let's say for office app to view various formats of docs.

This way either apps can have flavours according to usage of the file server.
